conf/routes

```diff
@@ -50,8 +50,9 @@ POST        /menuPerDayPerPerson/add           lunatech.lunchplanner.controllers
 GET         /menuPerDayPerPerson/attendees     lunatech.lunchplanner.controllers.MenuPerDayPerPersonController.getAttendeesEmailAddressesForUpcomingLunch
 
 # Rest API
-GET        /api/users                          lunatech.lunchplanner.controllers.RestController.getUser(email: String)
-GET        /api/events                         lunatech.lunchplanner.controllers.RestController.getFutureEvents(limit: Int ?= 2)
+GET         /api/authenticate                  lunatech.lunchplanner.controllers.RestController.validateAccessToken(accessToken: String)
```

Comment:
Shouldn't this be a POST instead? As it will return a new session every time right?

Now the access token can be easily seen in http access logs for example

Reply:
Since we're using https I think it's fine (in the sense that the parameters are encrypted). But you're right regarding the logs etc

Should I also make an issue to update the same route on the vacation app? Currently I think it's only used by the mobile prototype

Reply:
Yeah that would be a nice improvement I think. If you want you can merge this as a first step and improve later.

Reply:
I changed this one to POST already